### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by Snaked06**
Simply added the alt text for every images on the main site

# Issue #113 
Add alt text to images

### Changes
- added the alt text to images

### Flags
- No issue

### Related Issues
- Issue #113 
- Pull Request #113

### CODE REVIEW

**Comment by irmerk** [comment]
This is being worked on by a couple others in #113, will leave it open for a few days and review if those two haven't provided anything.

File: website/pages/en/index.js [hunk 1]
```diff
@@ -139,6 +139,7 @@ const Templates = props => (
       {
         content: "<div class='typeset'>Cicero templates are composed of three elements: the Template Grammar (the natural language text for the template), the Template Model (the data model that backs the template), the Logic (the executable business logic for the template). When combined these three elements allow templates to be edited, analyzed, queried and executed.</div>",
         image: imgUrl('../docs/assets/template.png'),
+        imageAlt: "Cicero templates composition using three elements: Model ,Grammer ,Logic",
```
Comment (jeromesimeon) [inline website/pages/en/index.js]: Please fix the English. Also I think a more concise alt text would be great, @Michael-Grover ?

File: website/pages/en/index.js [hunk 2]
```diff
@@ -153,6 +154,7 @@ const Grammar = props => (
       {
         content: '<div class="typeset">Create structured clauses and contracts by binding variables in natural language to a data model. Optionally, clauses can be made executable by adding contract logic to the data model.</div>',
         image: imgUrl('grammar.png'),
+        imageAlt: "Natural Language templates",
```
Comment (jeromesimeon) [inline website/pages/en/index.js]: Any reason why the different cases (`L` or `l` / `t` or `T` ?). Maybe use `Template` everywhere.

**Comment by Michael-Grover** [comment]
Someone else already claimed this issue before this PR was submitted and they have added alt text to this page, so I'm closing this PR.

**Pull request closed without merging**